rllib_contrib/a3c/README.rst

```diff
@@ -0,0 +1,21 @@
+A3C (Asynchronous Advantage Actor-Critic)
+-----------------------------------------
+
+`A3C <https://arxiv.org/abs/1602.01783>` is the asynchronous version of A2C, where gradients are computed on the workers directly after trajectory rollouts, and only then shipped to a central learner to accumulate these gradients on the central model. After the central model update, parameters are broadcast back to all workers. Similar to A2C, A3C scales to 16-32+ worker processes depending on the environment.
+
+
+Installation
+------------
+
+.. code-block:: bash
+
+    conda create -n rllib-a3c python=3.10
+    conda activate rllib-a3c
+    pip install -r requirements.txt
+    pip install -e '.[development]'
```

Comment:
some thought:
better UX: some people may not want to clone the repos.
AI (action item): add an alternative for installation that does the cloning behind the scene. 

```
conda create -n rllib-a3c python=3.10
conda activate rllib-a3c
pip install -r https://raw.githubusercontent.com/ray-project/ray/master/rllib/contrib/a3c/requirements.txt
pip install git+https://github.com/ray-project/ray.git#subdirectory=rllib_contrib/a3c
```

Reply:
I've added this for now, and we may need a separate pr to fix the links after we merge this pr.

Reply:
actually I'm going to open a separate pr for this lets merge as is